sklearn/metrics/cluster/_supervised.py

```diff
@@ -1033,12 +1033,16 @@ def adjusted_mutual_info_score(
         or classes.shape[0] == clusters.shape[0] == 0
     ):
         return 1.0
+    # if there is only one class or one cluster return 0.0.
+    elif classes.shape[0] == 1 or clusters.shape[0] == 1:
+        return 0.0
 
     contingency = contingency_matrix(labels_true, labels_pred, sparse=True)
     # Calculate the MI for the two clusterings
     mi = mutual_info_score(labels_true, labels_pred, contingency=contingency)
     # Calculate the expected value for the mutual information
     emi = expected_mutual_information(contingency, n_samples)
+    nominator = mi - emi
```

Comment:
Can you use `numerator` instead of `nominator`?

Not a native English speaker, but I never heard of `nominator` before and used `numerator` rings a bell. This seems to match this [StackExchange answer](https://math.stackexchange.com/a/159082).

Reply:
It was a mistake on my part, sorry